handlers/k8sconfig_handler.go

```diff
@@ -295,3 +328,65 @@ func (h *Handler) setupK8sConfig(inClusterConfig bool, k8sConfigBytes []byte, co
 	kc.ClusterConfigured = true
 	return kc, nil
 }
+
+// swagger:route PUT /api/system/kubernetes/context SystemAPI idPutKubernetesContext
+// Handle PUT request for changing Kubernetes Context
+//
+// Changes the current context of the in memory kubeconfig without interfering with the on-disk config file
+// responses:
+// 	200:
+
+// ChangeK8sContext - change k8s context
+func (h *Handler) ChangeK8sContext(w http.ResponseWriter, req *http.Request, prefObj *models.Preference, user *models.User, provider models.Provider) {
+	contextReqObj := map[string]interface{}{}
+
+	defer func() {
+		_ = req.Body.Close()
+	}()
+
+	if err := json.NewDecoder(req.Body).Decode(&contextReqObj); err != nil {
+		err = errors.Wrap(err, "unable to marshal the payload")
+		logrus.Error(ErrMarshal(err, "context-name"))
```

Comment:
Hey @utkarsh-pro I was thinking that we can be consistent with the `h.log.Error`.  Though I don't know the requirement here but I think if the usage is consistent then it would have been better I guess😃.

Reply:
Yep, you are right. Missed this out.